src/core/dash.h

```diff
@@ -213,7 +213,7 @@ class DashTable : public detail::DashTableBase {
   }
 
   size_t bucket_count() const {
-    return unique_segments_ * SegmentType::kNumBuckets;
+    return unique_segments_ * SegmentType::kRegularBucketCnt;
```

Comment:
wops, I think this should be unique_segments_ * kTotalBuckets

Reply:
he renamed numbuckets to regularbucketcnt, so it's not a functional change

Reply:
yes but it a bug that I noticed now when reviewing this code